students/nDruP/lesson03/seq_slicing.py

```diff
@@ -0,0 +1,75 @@
+def swap_first_last(seq):
+    """
+    Swap the first and last elements in a sequence
+    i.e. [1,2,3,4]->[4,2,3,1]
+    """
+    if len(seq)>1:
+        return seq[len(seq)-1:]+seq[1:len(seq)-1]+seq[:1]
+    return seq
```

Comment:
in your submission you asked how to deal with single element seq, that is exactly how I would do (what you have above)